variants/generator/input.svg

```diff
@@ -0,0 +1,1603 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This looks like the input for hundred specifically, shouldn't it live in the hundred directory?

Reply:
This is a good question. This is the input for Hundred, but my idea was that it could be used to provide the necessary layers.

Once the generator is run I expect the output map and go file to require manual tidying. The aim was to automate most of the effort.

Maybe I should replace this file with a basically empty svg template.

Reply:
Or just comment about why it looks like a hundred map :)